src/server/api/routers/project.ts

```diff
@@ -6,6 +6,12 @@ import { getUploadURL } from "@/server/s3";
 
 const allowedFileTypes = ["image/png", "image/jpeg"];
 const maxFileSize = 1048576; // 1MB
+const discordChannelTypes = [
+  0, // GUILD_TEXT
+  11, // PUBLIC_THREAD
+  12, // PRIVATE THREAD
```

Comment:
are these valid types? i dont think you can create a thread in a thread?

Reply:
resolved in 8f4cdf6425e28476bc8575eaa545bef6adc39a4b